doc/sphinx-guides/source/developers/dataset-migration-api.rst

```diff
@@ -0,0 +1,51 @@
+Dataset Migration API
+=====================
+
+The Dataverse software includes several ways to add Datasets originally created elsewhere (not to mention Harvesting capabilities). These include the Sword API (see the :doc:`/api/sword` guide) and the /dataverses/{id}/datasets/:import methods (json and ddi) (see the :doc:`/api/native-api` guide).
```

Comment:
We should mention that the DOI authority has to match. (That's my understanding anyway.)

Reply:
in line 9